string mqtt_topic
string raw_msg
int32 qos
